### DISCUSSION TITLE
how do I write a sensor that detects that op has failed and causes the whole job to fail?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17712] [Q&A]
<@U018K0G2Y85> How do I write a sensor that detects that op has failed and causes the whole job to fail?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C05M1ELAC07/p1699247838051249?thread_ts=1699247838.051249&cid=C05M1ELAC07).*

**Comment by sista05** [comment]
This discussion was not a problem, so it is OK to close it.
